index.js

```diff
@@ -21,4 +22,21 @@ function hideMenu() {
 
 document.getElementById('hamburger-id').addEventListener('click', showMenu);
 document.getElementById('cancel-id').addEventListener('click', hideMenu);
-document.querySelector('.menu-list').addEventListener('click', hideMenu);
\ No newline at end of file
+document.querySelector('.menu-list').addEventListener('click', hideMenu);
+
+// Email validatetion
+const myform = document.getElementById('myform');
+const email = document.getElementById('email');
+const error = document.querySelector('.error-msg');
+
+// the function below check if the email address is all lowercase or not //
+function emailCheckLowerCase(e) {
+  if (email.value !== email.value.toLowerCase()) {
+    error.classList.remove('hide');
+    e.preventDefault();
+  } else {
+    error.classList.add('hide');
+  }
+}
```

Comment:
- [ ] Hi great job. Please as mentioned in the requirements the lowercase validation is supposed to check, the email address when the user tries to submit the form.